source/images-guide.txt

```diff
@@ -61,13 +61,13 @@ itself should not be wider than 1000px.
    content.
 
    It may make sense to use
-   `mozjpeg <https://github.com/mozilla/mozjpeg>`_ if you need lossy
-   compression; `imagemagick <http://imagemagick.org/>`_ to resize images;
-   and a `Ninja <http://ninja-build.org/>`_ file to put the pieces
+   `mozjpeg <https://github.com/mozilla/mozjpeg>__ if you need lossy
```

Comment:
Issue: I think this is missing a closing backtick between the angled bracket and underscores.

Reply:
Good catch!